dataeng/resources/exporter-worker.sh

```diff
@@ -0,0 +1,14 @@
+TODAY=$(date +%d)
+
+env | sort
+
+${EXPORTER_VENV}/bin/exporter \
+   ${EXTRA_OPTIONS} ${CONFIG_PATH} ${ORG_CONFIG_PATH} \
+   --org=${ORG} \
+   --task=OrgEmailOptInTask \
+   --output-bucket=${OUTPUT_BUCKET} \
+   --output-prefix=${OUTPUT_PREFIX} \
+   --external-prefix=databases/${DATE:-$TODAY} \
```

Comment:
Just curious if this is actually needed...

Reply:
Went with what we have configure currently. 
Do we not need it anymore ?

Code:
https://github.com/edx/edx-analytics-exporter/blob/aec7c41332e3594ff6f884ecf2064d61a490b476/exporter/main.py#L29

Reply:
Hmmm...I'm not actually sure where this is used...I can't find it in the analytics exporter code or in the platform code.  I suspect it's unnecessary.  Also as an FYI, I have modifications to the analytics-exporter code currently in the zafft/testing branch.  Some of them will be carried forward (the removal of SUDO on all commands) and some of them will not be carried forward (the change of retries from 3 to 1).  We should resync this script before I make my PR.  As of right now there are no changes to any of the shell scripts.

Reply:
Oh, it's in the analytics exporter code, and is used by the CopyS3FileTask-derived task to copy CWSM data from the pipeline job output to include in the exports.  I'm just saying that it's not needed for the OrgEmail workflow, and it's there because of a copy-and-paste from the regular export.